.github/workflows/release_wheel.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/setup-python@v2
```

Comment:
Technically google org policy is now to pin to a specific version (`uses: actions/setup-python@7f80679172b057fc5e90d70d197929d454754a5a  # v2`), not a tagged release (`uses: actions/setup-python@v2`): https://opensource.google/documentation/reference/github/services#actions

I just updated the main IREE repo: https://github.com/google/iree/pull/8997 the other day. The risk area is smaller in this repo, but would still be nice to fix at some point.

Reply:
Done.